https://issues.redhat.com/browse/ACM-6365 (#5294)
open-cluster-management/rhacm-docs: release_notes/known_issues_application.adoc

// 2.6:25445

The `allow` and `deny` list feature does not work in Object storage application subscriptions.

[#No-longer-supporting-Argo-CD-pull-model]

> == Application console does not support Argo CD 

Reviewer: I would suggest changing this to:

`Application console does not support Argo CD pull model`

Otherwise it would sound like we don't support Argo CD at all which we do but just not the pull model.

// 2.8.6365

> The _Topology_  and _Console_ for _Application_ does not support displaying Argo CD applications deployed by the `Application Set` pull model.

Reviewer 2: I'm not sure why topology is mentioned. I don't see it in the issue. This is what was in the issue:

"UI does not support Argo CD pull model

ApplicationSet applications deployed using the Argo CD pull model will not display correctly in the topology as they are not supported currently."

**Console** does not support Argo CD pull model: I replaced UI with Console bc that is how it would be written in the product doc. Based on my change, it seems like the known issue should be in the Web console known issue. 

Is there a specific issue due to the ApplicationSet configuration/framework? 

```suggestion
When you deploy `ApplicationSet` applications using the Argo CD pull model, the application is displayed incorrectly from the _Topology_ page.
```

Reviewer: Nope, not an issue due to the ApplicationSet configuration/framework. It's because the ApplicationSet pull model is a new feature in ACM 2.8. The backend of the feature is done but we just didn't have the resources to work on the frontend/console part of the feature.

Author: I saw Topology in the Slack thread and believed it referred to the Topology UI as that what was in the message. 

Since it is not an issue with Applications, I'll go ahead and change the correct document, taking in the suggestions.

Reviewer 2: @faaden123 my bad, i definitely read over topolgy lol. Thanks for making the update
